conans/client/conf/__init__.py

```diff
@@ -80,7 +80,7 @@
                       "8", "8.1", "8.2", "8.3", "8.4",
                       "9", "9.1", "9.2", "9.3",
                       "10", "10.1", "10.2", "10.3",
-                      "11", "11.1", "11.2"]
+                      "11", "11.1", "11.2", "12"]
```

Comment:
Better put it in its own line, in anticipation of future minors
```suggestion
                      "11", "11.1", "11.2", 
                      "12"]
```